nclasses.asd

```diff
@@ -0,0 +1,35 @@
+;;;; SPDX-FileCopyrightText: Atlas Engineer LLC
+;;;; SPDX-License-Identifier: Public Domain
+
+(defsystem "nclasses"
+  :version "0.0.0"
+  :description "Simplify class like definitions with define-class and friends."
+  :author "dwim.hu & Atlas Engineer LLC"
```

Comment:
I'd rather split this into `:author "dwim.hu"` (or `:author "dwim.hu & Atlas Engineer LLC"`, given the scale of changes) and `:maintainer "Atlas Engineer LLC`.